Jenkinsfile

```diff
@@ -159,89 +159,103 @@ pipeline {
+        //                     bat "mingw32-make -j${env.PARALLEL} test-headers"
+        //                     setupCXX(false)
+        //                     runTestsWin("src/test/unit")
+        //             }
+        //             post { always { deleteDirWin() } }
+        //         }
+        //         stage('Linux Unit') {
+        //             agent { label 'linux' }
+        //             steps {
+        //                 unstash 'StanSetup'
+        //                 setupCXX(true, env.GCC)
+        //                 sh "g++ --version"
+        //                 runTests("src/test/unit")
+        //             }
+        //             post { always { deleteDir() } }
+        //         }
+        //         stage('Mac Unit') {
+        //             agent { label 'osx' }
+        //             steps {
+        //                 unstash 'StanSetup'
+        //                 setupCXX(false)
+        //                 runTests("src/test/unit")
+        //             }
+        //             post { always { deleteDir() } }
+        //         }
+        //     }
+        // }
         stage('Integration') {
             parallel {
-                stage('Integration Linux') {
-                    agent { label 'linux' }
-                    steps {
-                        unstash 'StanSetup'
-                        setupCXX(true, env.GCC)
-                        runTests("src/test/integration", separateMakeStep=false)
-                    }
-                    post { always { deleteDir() } }
-                }
-                stage('Integration Mac') {
-                    agent { label 'osx' }
+                // stage('Integration Linux') {
+                //     agent { label 'linux' }
+                //     steps {
+                //         unstash 'StanSetup'
+                //         setupCXX(true, env.GCC)
+                //         runTests("src/test/integration", separateMakeStep=false)
+                //     }
+                //     post { always { deleteDir() } }
+                // }
+                // stage('Integration Mac') {
+                //     agent { label 'osx' }
+                //     steps {
+                //         unstash 'StanSetup'
+                //         setupCXX()
+                //         runTests("src/test/integration", separateMakeStep=false)
+                //     }
+                //     post { always { deleteDir() } }
+                // }
+                stage('Math functions expressions test') {
+                    agent any
                     steps {
                         unstash 'StanSetup'
                         setupCXX()
-                        runTests("src/test/integration", separateMakeStep=false)
+                        script {
+                            withEnv(['PATH+TBB=./lib/stan_math/lib/tbb']) {
+                                try { bat "cd lib/stan_math; runTests.py -j${env.PARALLEL} test/expressions; cd .." }
```

Comment:
@serban-nicusor-toptal can I ask you for some help here. What would e the best way of running a Python script from the math folder here? I dont think cd-ing in works.